xml/smb_native_lm.xml

```diff
@@ -41,5 +41,21 @@
     <param pos="0" name="service.vendor" value="Samba"/>
     <param pos="0" name="service.product" value="Samba"/>
     <param pos="1" name="service.version"/>
+    <param pos="0" name="service.certainty" value="1.00"/>
   </fingerprint>
+
+  <!-- Windows -->
+  <fingerprint pattern="^(Windows [\W*\w*]*)$">
```

Comment:
I agree that this is a simpler approach rather than having one regular expression per flavor of Windows, but the problem with this (especially if you convert these to using `os.` rather than `service.`) is that you are taking the Windows banner as-is and asserting that as the product with 100% certainty.  There are numerous problems with this approach, the biggest / most common of which is the quality of the `.product` since you are believing whatever is there.